CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### v1.0.2 - 23/08/23
+
+- Bug correction.
```

Comment:
@FabricioLinkNacional É interessante dar detalhes do BUG que foi solucionado, ex.: "Fixed bug in custom posts headers built with Divi showing timestamp"